retriever/flags/flags.go

```diff
@@ -32,12 +32,13 @@ var (
 		Required: true,
 		EnvVar:   common.PrefixEnvVar(envPrefix, "TIMEOUT"),
 	}
-	BlsOperatorStateRetrieverFlag = cli.StringFlag{
-		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
-		Usage:    "Address of the BLS Operator State Retriever",
-		Required: true,
-		EnvVar:   common.PrefixEnvVar(envPrefix, "BLS_OPERATOR_STATE_RETRIVER"),
```

Comment:
don't remove this flag, since thats a breaking change. instead mark it as deprecated and log a warning when its used, prompting the user to use the new flags.